rotatescreen/display.py

```diff
@@ -44,41 +28,57 @@ def rotate_to(self, degrees):
         dm.DisplayOrientation = rotation_val
         win32api.ChangeDisplaySettingsEx(self.device, dm)
 
-    def set_landscape(self):
+    def set_landscape(self) -> None:
         self.rotate_to(0)
 
-    def set_landscape_flipped(self):
+    def set_landscape_flipped(self) -> None:
         self.rotate_to(180)
 
-    def set_portrait(self):
+    def set_portrait(self) -> None:
         self.rotate_to(90)
 
-    def set_portrait_flipped(self):
+    def set_portrait_flipped(self) -> None:
         self.rotate_to(270)
 
     @property
-    def current_orientation(self):
-        state = self.devicemodeW.DisplayOrientation
+    def current_orientation(self) -> int:
+        state: int = self.devicemodeW.DisplayOrientation
         return state * 90
 
     @property
-    def info(self):
+    def info(self) -> Dict:
         return win32api.GetMonitorInfo(self.hMonitor)
 
     @property
-    def device(self):
+    def device(self) -> str:
         return self.info["Device"]
 
     @property
-    def is_primary(self):
+    def is_primary(self) -> str:
```

Comment:
This isn't quite correct, `is_primary` uses the `MONITORINFOF_PRIMARY` which is an integer. This must be true to facilitate using this property for boolean checks.

```suggestion
    def is_primary(self) -> int:
```